pkg/controller/plan/kubevirt.go

```diff
@@ -1166,6 +1173,33 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 	return
 }
 
+// Attempt to find a suitable instancetype preference.
+func (r *KubeVirt) instancetype(vm *plan.VMStatus) (virtualMachine *cnv.VirtualMachine, ok bool) {
+	instancetypeName, err := r.Builder.InstanceTypeName(vm.Ref)
+	if err != nil {
+		return
+	}
+
+	instanceTypePreference := &instancetype.VirtualMachineClusterPreference{}
+	err = r.Destination.Client.Get(
+		context.TODO(),
+		client.ObjectKey{Name: instancetypeName},
+		instanceTypePreference)
+	if err != nil {
+		r.Log.Info("could not find instance type preference for destination VM.",
+			"vm",
+			vm.String(),
+			"error",
+			err)
+		return
+	}
+	virtualMachine = r.emptyVm(vm)
+	perMatcher := &cnv.PreferenceMatcher{Name: instanceTypePreference.Name}
```

Comment:
You need to set the `kind` if you want to use namespaced resources. Defaults to cluster-wide.